vue/src/Tag/TagEdit.vue

```diff
@@ -784,6 +785,13 @@ export default defineComponent({
     hasPublishCapability() {
       return Matomo.hasUserCapability('tagmanager_write') && Matomo.hasUserCapability('tagmanager_use_custom_templates');
     },
+    isTagHidden(id: string) {
```

Comment:
I'm suggesting a method name change since it appears to be returning the opposite of what the current name indicates.
```suggestion
    isTagVisible(id: string) {
```

Reply:
Updated as suggested :+1: